app/data/mwss_4.json

```diff
@@ -0,0 +1,3804 @@
+{
+    "data_version": "0.0.1",
+    "introduction": {
+        "description": "",
+        "information_to_provide": [
```

Comment:
We don't currently support extra content for the introduction page without creating another theme, @collisdigital is this required for the prototype?